iroha_data_model/src/events.rs

```diff
@@ -443,6 +449,16 @@ pub mod pipeline {
         pub hash: Hash,
     }
 
+    impl Default for Event {
+        fn default() -> Self {
+            Self {
+                entity_type: EntityType::Block,
+                status: Status::Committed,
+                hash: Hash([0; Hash::LENGTH]),
+            }
+        }
+    }
```

Comment:
Is it required somewhere? In general there is no default in this case. If there is no filter or incorrect filter - then we reject.